tests/Pest.php

```diff
@@ -1,3 +1,6 @@
 <?php
 
 uses()->group('integration')->in('Visual');
+uses()->beforeEach(function (): void {
```

Comment:
Since uses can also be used on the test files itself, it may be important to add a test for that too.

Reply:
What do you think is the best behaviour in this case? We could have the test's `use` call overwrite the global one in `Pest.php`, **_or;_** compose them and have the global execute first, then the local? Perhaps something else?

Reply:
Here is an example, where I dump the order of execution and I the global + local `uses` get composed

![image](https://user-images.githubusercontent.com/18744334/113784973-dcc83c80-96fb-11eb-86d4-3aae52127eab.png)

![image](https://user-images.githubusercontent.com/18744334/113785029-f79ab100-96fb-11eb-8ce2-7f72465c36b7.png)

Reply:
Compose them. First runs the global, after that runs the local one in the file.

Reply:
Yeah - your example is perfect.